javascript-if/main.js

```diff
@@ -0,0 +1,61 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(num) {
+  return num < 5;
+}
+
+function isEven(num) {
+  return (num % 2 === 0);
+}
+
+function startsWithJ(string) {
+  return string[0] === 'J' || string[0] === 'j';
+}
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+function isOldEnoughToDrive(person) {
+  return person.age >= 16;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return 'false - this is illegal';
+}
+
+function categorizeAcidity(pH) {
+  if (pH > 0 && pH < 7) {
+    return 'acid';
+  } else {
+    if (pH === 7) {
+      return 'neutral';
+    } else {
+      if (pH > 7 && pH < 14) {
+        return 'base';
+      } else {
+        if (pH < 0 || pH > 14) {
+          return 'invalid pH level';
+        }
+      }
+    }
+  }
+}
```

Comment:
What happens if `pH === 0` or `pH === 14`? Please make sure your function accounts for those possibilities:

---

![image](https://user-images.githubusercontent.com/16143771/152251352-540a6c06-d115-40e7-b7b0-8d5f1c152a71.png)